### PR TITLE
Ankit | Test | Changes in Custom Email under invoice setting

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cx2uwa3

## Release Notes

- **New Features**
	- Enhanced email input sections with improved styling and layout.
	- New input property `chipPrefix` for customizable chip prefixes in the select multiple fields component.
	- Automatic focus on the input field upon component initialization.

- **Bug Fixes**
	- Improved conditional rendering for CC and BCC sections to ensure consistent display.

- **Style**
	- Updated styles for `.chiplist-error` and `.select-multiple-field-input` classes.
	- Removed the `.border-bottom` class for simplified styling.

### CODE REVIEW

File: apps/web-giddh/src/app/shared/template-froala/template-froala.component.html [hunk 1]
```diff
@@ -14,68 +14,76 @@ <h3>{{ commonLocaleData?.app_custom_email_template }}</h3>
             <form [formGroup]="emailForm" class="d-flex flex-column flex-grow-1">
                 <div class="chip-gap" (clickOutside)="clickedOutsideEmail()" (click)="clickedInsideEmail()">
                     <div class="form-group position-relative">
-                        <div class="d-flex">
+                        <div class="d-flex communication-input">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/template-froala/template-froala.component.html]:
```suggestion
                        <div class="d-flex border-bottom">
```

File: apps/web-giddh/src/app/shared/template-froala/template-froala.component.scss [hunk 2]
```diff
@@ -15,8 +15,12 @@
...
-    border-radius: 6px; 
-    padding: 2px 4px; 
-    color: var(--color-common-primary-text); 
+    border-radius: 6px;
+    padding: 2px 4px;
+    color: var(--color-common-primary-text);
     box-shadow: inset 0 0 0 1px var(--color-warning);
+}
+
+.communication-input {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/template-froala/template-froala.component.scss]: remove

File: apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts [hunk 3]
```diff
@@ -29,6 +29,8 @@ export class SelectMultipleFieldsComponent implements OnInit, OnDestroy, OnChang
     @Input() public validations: any[] = [];
     /** CSS class name to add on the field */
     @Input() public showError: boolean = false;
+    /** Holds prefix of chip */
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts]:
```suggestion
    /** Holds prefix of chip text  */
```

File: apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts [hunk 4]
```diff
@@ -98,6 +100,19 @@ export class SelectMultipleFieldsComponent implements OnInit, OnDestroy, OnChang
...
+     * This function puts the focus in input
+     *
+     * @memberof SelectMultipleFieldsComponent
+     */
+    public ngAfterViewInit(): void {
+        setTimeout(() => {
+            if (this.selectField) {
+                this.selectField.nativeElement.focus();
+            }
+        },100);
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts]:
```suggestion
        }, 100);
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/template-froala/template-froala.component.html (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/template-froala/template-froala.component.html]: Done

File: apps/web-giddh/src/app/shared/template-froala/template-froala.component.scss (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/template-froala/template-froala.component.scss]: Done

File: apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts]: Done

File: apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts (hunk 4 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.ts]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**